site2/docs/security-kerberos.md

```diff
@@ -21,7 +21,9 @@ If your organization is already using a Kerberos server (for example, by using `
 
 Note that if you are using Oracle Java, you need to download JCE policy files for your Java version and copy them to the `$JAVA_HOME/jre/lib/security` directory.
 
-#### Kerberos Principals
+Here is some other prerequisites that will use in this configuration.
```

Comment:
```suggestion

```

Consider removing this sentence? 

Reasons:
1. Based on the context, it does not need a transition sentence here.
![image](https://user-images.githubusercontent.com/50226895/61676868-142c6500-ad30-11e9-92c6-db7962af51c8.png)

2. If the sentence is removed, it has no impact on the doc.